build.html

```diff
@@ -9,24 +9,24 @@ <h1 class="text-center">Login</h1>
       <form class="form-horizontal form-login">
         <div class="form-group clearfix ">
           <div class="col-sm-12 fl-email">
-            <input type="email" class="form-control profile_email" name="Email" placeholder="Enter your email" autocomplete="false" />
+            <input type="email" class="form-control profile_email outline-item-selector" name="Email" placeholder="Enter your email" autocomplete="false" />
```

Comment:
The word `selector` is redundant. Use this class instead.

```suggestion
            <input type="email" class="form-control profile_email focus-outline" name="Email" placeholder="Enter your email" autocomplete="false" />
```

Reply:
Fixed